bin/desi_qso_qn_afterburner

```diff
@@ -429,22 +435,22 @@ if __name__ == "__main__":
                 sys.exit(1)
 
             log.info(f"Nbr objetcs for QN: {sel_to_QN.sum()}")
-            QSO_from_QN = selection_targets_with_QN(redrock, fibermap, sel_to_QN,
-                    DESI_TARGET, args.coadd, param_QN, param_RR, args.save_target)
+            QSO_from_QN = selection_targets_with_QN(redrock, fibermap, sel_to_QN, DESI_TARGET,
+                                                    args.coadd, param_QN, param_RR, args.save_target)
 
             if QSO_from_QN.shape[0] > 0:
                 log.info(f"Number of targets saved : {QSO_from_QN.shape[0]} -- "
                          f"Selected with QN + new RR: {QSO_from_QN['IS_QSO_QN_NEW_RR'].sum()}")
                 save_dataframe_to_fits(QSO_from_QN, args.output, DESI_TARGET)
             else:
-                file = open(os.path.splitext(args.output)[0]+'.notargets', "w")
+                file = open(os.path.splitext(args.output)[0] + '.notargets', "w")
```

Comment:
While we're making changes, please update this to be '.notargets.txt' to give a hint about the filetype (like Ben did for '.misscamera.txt')